pkg/virt-launcher/virtwrap/access-credentials/access_credentials.go

```diff
@@ -310,7 +305,7 @@ func (l *AccessCredentialManager) pingAgent(domName string) error {
 }
 
 func (l *AccessCredentialManager) agentSetAuthorizedKeys(domName string, user string, authorizedKeys []string) error {
-	err := func() error {
+	firstErr := func() error {
```

Comment:
nit: Using `err` is just fine as I am not sure we gain anything more with `firstErr`

Reply:
done